app/frontend/src/components/EditLocationMap.tsx

```diff
@@ -100,78 +101,72 @@ export default function EditLocationMap({
     if (e.type === "touchstart") {
       const handleTouchMove = (e: MapTouchEvent) => onCircleMove(e);
       map.current!.on("touchmove", handleTouchMove);
-      map.current!.once("touchend", (e) => onCircleUp(e, handleTouchMove));
+      map.current!.once("touchend", (e) =>
+        handleCoordinateMoved(e, handleTouchMove)
+      );
     } else {
       const handleMove = (e: MapMouseEvent) => onCircleMove(e);
       map.current!.on("mousemove", handleMove);
-      map.current!.once("mouseup", (e) => onCircleUp(e, handleMove));
+      map.current!.once("mouseup", (e) => handleCoordinateMoved(e, handleMove));
     }
   };
 
   const onCircleMove = (e: MapMouseEvent | MapTouchEvent) => {
-    centerCoords.current = e.lngLat.wrap();
+    const wrapped = e.lngLat.wrap();
+    commit(
+      {
+        lat: wrapped.lat,
+        lng: wrapped.lng,
+      },
+      false
+    );
     redrawMap();
   };
 
-  const onCircleUp = (
+  const handleCoordinateMoved = (
     e: MapMouseEvent | MapTouchEvent,
-    moveHandler: (x: any) => void
+    moveHandler: (x: any) => void = () => null
```

Comment:
What???

Reply:
What what?

Reply:
Why was this line changed? I don't even understand what this type is :p

Reply:
`moveHandler` is a function with sig `(x: any) => void`, and it's default value is `() => null`: like half the usages weren't using the `moveHandler` so I just added a default